wsd/COOLWSD.cpp

```diff
@@ -2690,9 +2690,11 @@ void COOLWSD::setLogLevelsOfKits(const std::string& level)
 void PrisonPoll::wakeupHook()
 {
 #if !MOBILEAPP
+    THREAD_UNSAFE_DUMP_BEGIN
```

Comment:
I can live with this, though this isn't really a dump state in a signal handler, right?

If so, perhaps it would be better to use Log::isEnabled() to check if we have trace-level logging enabled and use a mutex in that case to get DocBrokers::size without a mutex?